libs/platform/unit/platform_unit_test.cpp

```diff
@@ -440,6 +441,73 @@ test_provider_attach_client(
     return STATUS_SUCCESS;
 };
 
+static NTSTATUS
+test_denied_client_attach_provider(
```

Comment:
consider renaming to `test_denied_provider_attach_client`

Reply:
Modified.